.rhcicd/clowdapp-aggregator.yaml

```diff
@@ -61,6 +61,8 @@ objects:
           value: https://39032857c8214cedaf226c4a26709b5b@o271843.ingest.sentry.io/5987783?environment=${ENV_NAME}
         - name: QUARKUS_LOG_SENTRY_ENVIRONMENT
           value: ${ENV_NAME}
+        - name: PROMETHEUS_PUSH_GATEWAY
+        - value: ${PROMETHEUS_PUSHGATEWAY}
```

Comment:
@gwenneg is this correct? How do I set the PROMETHEUS_PUSHGATEWAY variable in clowder for each environment (stage, prod, ..) ?

Reply:
- There is a bogus `-` in front of `values`
- If you map the container parameter PROMETHEUS_PUSHGATEWAY to PROMETHEUS_PUSH_GATEWAY environment parameter you also need to declare the container parameter in the parameters section lower in that same file.

Reply:
> * If you map the container parameter PROMETHEUS_PUSHGATEWAY to PROMETHEUS_PUSH_GATEWAY environment parameter you also need to declare the container parameter in the parameters section lower in that same file.

That's the problem here. I was able to find the the value of PROMETHEUS_PUSHGATEWAY somewhere deep in app-interface and wasn't sure if it's a good idea to copy and paste it in this file. I acutally thought I could get the value somehow with clowder-magic in the container/application.

Reply:
You just reference the name, not the value, the value will be picked from the environment file:
`data/products/insights/environments/production.yml` `PROMETHEUS_PUSHGATEWAY: blah`

It's exactly the same as for `ENV_NAME`

Reply:
Thanks. I didn't see there is an entry without value.